### ISSUE TITLE
[Sandboxed Google Play] Delete Advertising ID by default

### ISSUE THREAD

**Issue by celenityy** [GrapheneOS/os-issue-tracker#3741]
I've noticed that fresh installs of Sandboxed Play Services seem to create an Advertising ID by default. A lot of users may not be aware of this or think to check their settings. I think it'd be a good idea to delete the ID by default, due to the privacy concerns it poses.

**Comment by Retold3202** [comment]
> Apps can simply create their own random number too.

True, I thought of this as well, but I don't think they're actively doing so, most seem to be relying on Google's Advertising ID & actively using it for tracking, which makes sense since they can generally count on it always being installed on the majority of Android devices.

**Comment by MichaelDevon** [comment]
Good idea. This would be equivalent of Apple's "Ask app not to track".
